pages/about.js

```diff
@@ -97,16 +104,45 @@ about.defaultProps = {
         'Associate Professor, University of Toronto',
       ],
     },
-    three: {
+    {
       name: 'Dr. Josephine Wong',
       titles: [
         'Professor and Research Chair in Urban Health',
         'Ryerson University',
       ],
     },
-    four: {
+    {
       name: 'Dr. Todd Girard',
       titles: ['Associate Professor', 'Ryerson University'],
     },
-  },
+  ],
+  researchers: [
+    {
+      name: 'Julia Gervaslo',
+      titles: ['MA student, Ryerson University'],
+    },
+  ],
+  techSupport: [
+    {
+      name: 'Likeable IT Inc',
+      titles: [],
+    },
+    {
+      name: 'Cache & Cookies',
+      titles: ['Software Development'],
+    },
+  ],
+  funderImages: [
+    {
+      organization: 'rbc',
+      url: '/images/RBC.png',
+      caption:
+        'The Royal Bank of Canada’s Partnership for Change, Inclusion, and Diversity Project Grant',
+    },
+    {
+      organization: 'cpa',
+      url: '/images/CPA.jpg',
+      caption: 'Canadian Psychological Association Student Research Grant',
+    },
+  ],
```

Comment:
Is there a reason why this is done in the pages/ default props then you pass it in the components rather than doing the component's default props?

Would love to hear your thoughts?


### Related Thread
https://github.com/Cache-and-Cookies/msmr/pull/59#discussion_r579860706

Reply:
wait wait @jimmyygit  @LemonFace0309 
 i think i this case it makes sense to have the copy data in the `pages/` cuz `Achievements` is being called multiple times with different copy. unless he makes a another component to wrap the 3 achievement components, which is probably best to do a wrapper

mb

Reply:
nws